modules/gce_remote_docker/daemon-direct-lvm.json

```diff
@@ -17,5 +17,6 @@
   "tlsverify": true,
   "tlscacert": "/etc/docker/ca.pem",
   "tlscert": "/etc/docker/server-cert.pem",
-  "tlskey": "/etc/docker/server-key.pem"
+  "tlskey": "/etc/docker/server-key.pem",
+  "registry-mirrors": ["staging-1-docker-registry.gce-us-central1.travisci.net"]
```

Comment:
I assume this should not be hardcoded :grin:  Maybe the generated config in `tfw` should account for registry mirrors: https://github.com/travis-ci/tfw/blob/bfbffd80e1ddbcfd13cf58b9fbe435949e810d3a/bin/tfw#L1270-L1288

Reply:
Ahhh I was looking all over for this. Thanks